tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -1509,7 +1509,7 @@ def test_limit_without_orderby_excess_groups_pruned(self):
                 {
                     "org_id": org_id,
                     "project_id": self.project.id,
-                    "metric_id": self.transaction_lcp_metric,
+                    "metric_id": rh_indexer_record(org_id, SessionMRI.HEALTHY.value),
```

Comment:
This test is incorrect, you are adding healthy session counts to distribution metrics, please fix this test
How about instead of deleting the transaaction measurement lcp and so on, we just split this test into two, one for session fields and one for perfromance